src/test/resources/testPrograms/patternMatching/CaseClasses.monty

```diff
@@ -0,0 +1,62 @@
+println("--- haskell-style lists ---")
+
+abstract class List:
+    + Int getLength():
+        case self of:
+            Cons(Int value, List l):
```

Comment:
So the compiler does not enforce all variants of the case class to be matched? Or at least via a wildcard?